### DISCUSSION TITLE
LORAs and weights supported?

### DISCUSSION THREAD

**Discussion by jszeck** [adieyal/sd-dynamic-prompts#274] [Q&A]
<my-lora:{.1|0.25|.5|.7|.9|1.0}>
or
{<loraA:.1>| <loraA:.5> | etc }
and 
{<loraB:.5>| <loraC:.8> | etc }
?

Are Lora's even supported? I'm getting strange outputs. almost like the prompt is being ignored sometimes but not others?
Also it's taking 2-3x time per step to generate the image? (I have like 10 wildcards in the prompt with 3-8 options each, and about 20 LORAs). 

![image](https://user-images.githubusercontent.com/17091777/221354583-1db09efa-9377-4dcc-9b36-3ae7c53215e9.png)

**Comment by Kadah** [comment]
The LoRA(s) in the first prompt will apply to the entire batch as each image in a batch cannot have different models.

Use batch size 1 and increase batch count >1.
